roxie/ccd/ccdmain.cpp

```diff
@@ -742,7 +742,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
         else
         {
             Owned<ILoadedDllEntry> dll = createExeDllEntry(argv[0]);
-            if (checkEmbeddedWorkUnitXML(dll))
+            if (containsEmbeddedWorkUnit(dll))
```

Comment:
This looked like it only picked up old ones to me...

Reply:
looks like this has been addressed (containsEmbeddedWorkUnit now checks both )